src/main/java/oreo/task/Event.java

```diff
@@ -0,0 +1,57 @@
+package oreo.task;
+
+import oreo.datetime.TimeParser;
+import oreo.exception.IllegalDateTimeException;
+
+public class Event extends Task {
+    private String fromDate;
+    private String fromTime;
+    private String toDate;
+    private String toTime;
+    public Event(String d, String fromDate, String fromTime,
```

Comment:
Shouldn't the d be named into a more straight forward variable? I have seen this issue in many other places too.